developer-docs-site/docs/concepts/basics-txns-states.md

```diff
@@ -67,3 +67,15 @@ A [signed transaction](/guides/sign-a-transaction.md) on the blockchain contains
 - **Maximum gas amount**: The [maximum gas amount](/concepts/basics-gas-txn-fee#gas-and-transaction-fee-on-the-aptos-blockchain) is the maximum gas units the transaction is allowed to consume.
 - **Sequence number**: This is an unsigned integer that must be equal to the sender's account [sequence number](/concepts/basics-accounts#account-sequence-number) at the time of execution.
 - **Expiration time**: A timestamp after which the transaction ceases to be valid (i.e., expires).
+
+## Proof
+
+The Aptos blockchain uses proof to verify the authenticity and correctness of the blockchain data.
```

Comment:
These should say `Proofs` since lower down we refer to them as `Cryptographic proofs`